addons/voice/org.openhab.voice.marytts/src/main/java/org/openhab/voice/marytts/internal/MaryTTSAudioStream.java

```diff
@@ -89,7 +89,7 @@ private InputStream getWavHeaderInputStream(int length) throws IOException {
         byte format = 0x10; // PCM
         byte bits = 16;
         byte channel = 1;
-        int srate = 48000;
+        long srate = (this.audioFormat != null) ? this.audioFormat.getFrequency() : 16000l;
```

Comment:
There are 3 voices within the default bundle, 1 en_US, 2 de_DE
Only the US uses 48kHz, so it is 1:2 for 16kHz
Anyway us voice is working well, because for real text synthetics, the audioFormat is set correctly. Only the MaryTTSService.initAudioFormats() need a default value.